utils/parametric/spec/trace.py

```diff
@@ -227,6 +239,29 @@ def find_span_in_traces(traces: List[Trace], span: Span) -> Span:
     return max_similarity_span
 
 
+def find_span_in_traces(traces: List[Trace], span: Span) -> Span:
+    """Return a span from the traces that matches all fields in `span`."""
+    return _assert_span_match(span, find_similar_span_in_traces(traces, span))
+
+
+def _assert_span_match(span: Span, similar: Span) -> Span:
+    for var in Span.__annotations__:
+        if not var.startswith("__"):
+            val = span.get(var)
+            s_val = similar.get(var)
+            if val != None and val != s_val:
+                # TODO remove this special handling once nodejs sets service in the same way.
+                # Right now, nodejs sets a tag named "service" in meta with the correct value.
```

Comment:
There is some special handling for `nodejs` here based on the discussion in `slack` about a difference in behavior